src/IW/Sync/Update.hs

```diff
@@ -0,0 +1,25 @@
+module IW.Sync.Update where
+
+import Control.Monad.IO.Unlift (MonadUnliftIO)
+import UnliftIO.Async (mapConcurrently_)
+
+import IW.App (WithError)
+import IW.Db
+import IW.Effects.Cabal (MonadCabal (..))
+import IW.Sync.Search
+
+
+fetchAndUpsertRepos
+    ::
+    ( MonadCabal m
```

Comment:
According to our style guide, we shouldn't have hanging `::`. This code should be formatted like this:

```haskell
fetchAndUpsertRepos
    :: ( MonadCabal m
       , MonadUnliftIO m
       ...
       )
    => m ()
```

Reply:
@chshersh Right!